samples/src/Cache/CacheAspect.cs

```diff
@@ -57,6 +57,6 @@ public class CacheAspect
         }
 
         protected string GetKey(MethodInfo method, IEnumerable<object> args) =>
-            $"{method.GetHashCode()}-{string.Join("-", args.Select(a => a.GetHashCode()))}";
+            $"{method.GetHashCode()}_{string.Join("_", args.Select(a => a == null ? "null" : a.GetHashCode().ToString()))}";

```

Comment:
Could you please use NullMarker.GetHashCode() instead of "null"
https://github.com/pamidur/aspect-injector/blob/7b50540b0d5aa9d777384f802593178ff2996cf2/samples/src/Cache/CacheAspect.cs#L12